frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -59,12 +74,19 @@ onUnmounted(() => {
 
 const onApplicationSelected = async (e: DropdownChangeEvent) => {
     setSelectedApplication(e.value ? JSON.stringify(e.value) : null);
+
     if (e.value.id === FAM_APPLICATION_ID) {
         applicationAdmins.value = await fetchApplicationAdmins();
+        // make the first tab active
+        resetActiveTab.value = 0;
```

Comment:
this is good, but this should always happen, no matter if the selected app is FAM or not, so let's move it outside the if else condition

Reply:
sure, I'll make the change